src/bandersnatch/mirror.py

```diff
@@ -208,7 +208,10 @@ def __init__(
             self.storage_backend = next(iter(storage_backend_plugins()))
         self.stop_on_error = stop_on_error
         self.loop = asyncio.get_event_loop()
-        self.homedir = self.storage_backend.PATH_BACKEND(str(homedir))
+        if isinstance(homedir, WindowsPath):
+            self.homedir = self.storage_backend.PATH_BACKEND(homedir.as_posix())
+        else:
+            self.homedir = self.storage_backend.PATH_BACKEND(str(homedir))
```

Comment:
I would expect path lib to work this out ... And isn't the logic backwards here - If WindowsPath use posix ?

Reply:
things would go wrong when running this program in Windows. As the homedir is a `Path` instance , when it runs on windows , `str(Path('/bucket/path'))` would result in something like `C:\bucket\path`, this is correct in windows but will get into trouble when init S3Path instance, but maybe I can handle it inside the StorageClass